frontend/package.json

```diff
@@ -4,9 +4,6 @@
   "private": true,
   "proxy": "http://localhost:3001",
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
If those testing libraries are not for production, better put them inside dev dependencies

Reply:
this library is created automatically by 'create-react-app'

Reply:
Great, I didn't know that  
This is really intreating, it's seems like "create react app" has a rule that they put every package inside dependencies and don't use devDependencies at all